sources/api/schnauzer/src/helpers.rs

```diff
@@ -1509,6 +1559,18 @@ fn oci_spec_resource_limits(value: &Value) -> Result<String, RenderError> {
 
 // =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=
 // helpers to the helpers
+/// Gets the runtime for the string.
+fn get_runtime(runtime: &str) -> Result<Runtime, RenderError> {
+    match runtime {
+        "docker" => Ok(Runtime::Docker),
+        "containerd" => Ok(Runtime::Containerd),
+        _ => Err(RenderError::from(
+            error::TemplateHelperError::InvalidVariant {
+                var: runtime.to_owned(),
+            },
+        )),
+    }
+}
```

Comment:
You should be able to deserialize directly from the input into this type; `serde_plain` can help.